Library/Formula/walkmod.rb

```diff
@@ -0,0 +1,20 @@
+class Walkmod < Formula
+  desc "Java-based project to apply and share code conventions"
+  homepage "http://www.walkmod.com"
+  url "https://bitbucket.org/rpau/walkmod/downloads/walkmod-1.3.0-installer.zip"
+  sha256 "01ea47c99f7d7bffeb0cd0d960fdab4f09a7d51a1e0979c6b4ff71d0aeb049fd"
+  depends_on :java
```

Comment:
Does it need a specific version? Also leave an empty line between the `sha` and the first `depends_on`, Thanks.